### PR TITLE
Adds Dominian assault rifle

### PR DESCRIPTION
**Pull request by BLUNTFORCE420**
See above, statistical copy-paste of 762 AR along the lines of other factional rifles, for eventual Dominian ship PR.

image

### CODE REVIEW

**Comment by BLUNTFORCE420** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Review by KingOfThePing — APPROVED** [review]

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 1]
```diff
@@ -253,6 +253,22 @@
 /datum/firemode/z8
 	var/use_launcher = 0
 
+/obj/item/gun/projectile/automatic/rifle/dom
+	name = "dominian assault rifle"
+	desc = "The standard-issue rifle of the Empire of Dominia’s Imperial Army. With its sleek lines and ruggedized faux-wood furniture, the MPR-24/5 \
+	is as elegant as it is deadly, proving itself in service during Dominia's numerous colonial conflicts."
+	desc_fluff = "The Moroz Pattern Rifle, Year of 2424 has been the standard-issue rifle of the Imperial Army for decades thanks to its durable construction and ease of use. \
```
Comment (SleepyGemmy) [inline code/modules/projectiles/guns/projectile/automatic.dm:260]:
```suggestion
	desc_extended = "The Moroz Pattern Rifle, Year of 2424 has been the standard-issue rifle of the Imperial Army for decades thanks to its durable construction and ease of use. \
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 2]
```diff
@@ -253,6 +253,22 @@
 /datum/firemode/z8
 	var/use_launcher = 0
 
+/obj/item/gun/projectile/automatic/rifle/dom
+	name = "dominian assault rifle"
+	desc = "The standard-issue rifle of the Empire of Dominia’s Imperial Army. With its sleek lines and ruggedized faux-wood furniture, the MPR-24/5 \
```
Comment (SleepyGemmy) [inline code/modules/projectiles/guns/projectile/automatic.dm:258]:
```suggestion
	desc = "The standard issue rifle of the Empire of Dominia’s Imperial Army. With its sleek lines and ruggedized faux wood furniture, the MPR-24/5 \
```

File: code/modules/projectiles/guns/projectile/automatic.dm (hunk 1 above)
Comment (SleepyGemmy) [inline code/modules/projectiles/guns/projectile/automatic.dm:260]:
```suggestion
	desc_fluff = "The Moroz Pattern Rifle, Year of 2424 has been the standard issue rifle of the Imperial Army for decades thanks to its durable construction and ease of use. \
```

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 3]
```diff
@@ -253,6 +253,22 @@
...
+	is as elegant as it is deadly, proving itself in service during Dominia's numerous colonial conflicts."
+	desc_fluff = "The Moroz Pattern Rifle, Year of 2424 has been the standard-issue rifle of the Imperial Army for decades thanks to its durable construction and ease of use. \
+	The Imperial Army has repeatedly modernized and updated the MPR-24 over the past 40 years, and the rifle is now in its fifth modernization: one that they hope will keep it competitive well into the 2470s."
+	icon = 'icons/obj/guns/dom_rifle.dmi'
+	icon_state = "acr"
+	item_state = "acr"
+	auto_eject = 1
```
Comment (SleepyGemmy) [inline code/modules/projectiles/guns/projectile/automatic.dm:265]:
```suggestion
	auto_eject = TRUE
```

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 4]
```diff
@@ -253,6 +253,22 @@
 /datum/firemode/z8
 	var/use_launcher = 0
 
+/obj/item/gun/projectile/automatic/rifle/dom
+	name = "dominian assault rifle"
+	desc = "The standard issue rifle of the Empire of Dominia’s Imperial Army. With its sleek lines and ruggedized faux wood furniture, the MPR-24/5 \
+	is as elegant as it is deadly, proving itself in service during Dominia's numerous colonial conflicts."
+	desc_extended = "The Moroz Pattern Rifle, Year of 2424 has been the standard-issue rifle of the Imperial Army for decades thanks to its durable construction and ease of use. \
```
Comment (SleepyGemmy) [inline code/modules/projectiles/guns/projectile/automatic.dm:260]:
```suggestion
	desc_extended = "The Moroz Pattern Rifle, Year of 2424 has been the standard issue rifle of the Imperial Army for decades thanks to its durable construction and ease of use. \
```

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**